### PR TITLE
DUOS-337 - MultiDAC: DUOS Dataset Upload Form (Mockups)

### PR DESCRIPTION
**Pull request by jaime-bernales-broad**
[JIRA Ticket here](https://broadinstitute.atlassian.net/browse/DUOS-337)

**To reproduce:**
In Admin Console > Click in "Add Datasets" section/button
In Datasets Modal > Click in "Upload a .cvs file"

**What functionalities are pending?**
Once you click "Upload a .cvs file":
- "Cancel" button must change the label to "Back" (design reference is on the ticket)
- Then, if you click "Cancel" the state "uploadForm" must change to affect the isRendered. There is a function created to toggle states that's called "switchUploadForm()". Same validations must run for Submit and Cancel buttons

### CODE REVIEW

File: src/components/modals/AddDacModal.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+import { Alert } from '../Alert';
+import AsyncSelect from 'react-select/lib/Async';
+
+
+export const AddDacModal = hh(class AddDacModal extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isEditMode: false,
```
Comment (rushtong) [inline src/components/modals/AddDacModal.js:13]: Curious, why not pull this from props by default?

File: src/components/Alert.css [hunk 2]
```diff
@@ -53,3 +53,81 @@
...
+}
+
+.checkbox.overwrite {
+  margin: 0 0 68px 0;
+  width:100%;
+}
+
+.checkbox.overwrite .regular-checkbox {
+  color: #000000;
+}
```
Comment (rushtong) [inline src/components/Alert.css]: nit: newline

**Review by rushtong — COMMENTED** [review]
Looks good at a high level. When back end support for this is in place, we can revisit the implementation.

File: src/components/modals/AddDacModal.js (hunk 1 above)
Comment (jaime-bernales-broad) [reply src/components/modals/AddDacModal.js:13]: I think is doing it here:

https://github.com/DataBiosphere/duos-ui/pull/170/files/6c63459275e9c4ef8d315c5cc9e66ddfce261d68#diff-bc379c6c14338ead4fd2da1117230aa8R32

Comment (rushtong) [reply src/components/modals/AddDacModal.js:13]: That happens *after* the component mounts. You already know what the edit mode is when this component is created ... it's in the passed in props. Why set it to false, wait for the component to mount, then change it? You already know what it is right here in the constructor. 

99% of the time, my goal is to reduce the complexity. More complexity === more bugs.

File: src/components/Alert.css (hunk 2 above)
Comment (jaime-bernales-broad) [reply src/components/Alert.css]: @rushtong branch updated with this change

File: src/components/modals/AddDacModal.js (hunk 1 above)
Comment (jaime-bernales-broad) [reply src/components/modals/AddDacModal.js:13]: @rushtong ok, I can't resolve this. I had been asking for some help these days, but I can't do more than this. Sorry :-(

Comment (rushtong) [reply src/components/modals/AddDacModal.js:13]: Thank you, no worries - I'll pick it up from here.

**Comment by rushtong** [comment]
Closing this for now. 
* The PR title and description are are confusing. There are some fixes for dataset uploads, but there is also UI work for https://broadinstitute.atlassian.net/browse/DUOS-354, not DUOS-337

**Pull request closed without merging**